plugins/codemodder-plugin-maven/src/test/java/io/codemodder/plugins/maven/operator/POMOperatorVersionQueryTest.java

```
@@ -20,13 +19,60 @@ final class POMOperatorVersionQueryTest {
   void testBasicQuery() {
     String pomFile = "pom-1.xml";
 
-    Arrays.stream(QueryType.values())
-        .filter(queryType -> queryType != QueryType.NONE)
+    Optional<VersionQueryResponse> optionalVersionQueryResponse = null;
+    try {
+      optionalVersionQueryResponse = versionDefinitions(pomFile);
+    } catch (DocumentException e) {
+      throw new RuntimeException(e);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(e);
+    } catch (XMLStreamException e) {
+      throw new RuntimeException(e);
+    }
+
+    LOGGER.debug("Versions found: {}", optionalVersionQueryResponse);
+
+    VersionQueryResponse versionQueryResponse = optionalVersionQueryResponse.get();
+
+    Assert.assertTrue(
+        "Version defined is 1.8 as source", versionQueryResponse.getSource().satisfies("=1.8.0"));
+    Assert.assertTrue(
+        "Version defined is 1.8 as target", versionQueryResponse.getTarget().satisfies("=1.8.0"));
+  }
+
+  @Test
```

On second thought, why are we catching any of these? Just let them propagate and fail the test.